builtin/mainmenu/dlg_create_world.lua

```diff
@@ -353,6 +353,14 @@ end
 
 local function create_world_buttonhandler(this, fields)
 
+	if fields["world_create_open_cdb"] then
+		local dlg = create_store_dlg("game")
+		dlg:set_parent(this)
+		this:hide()
```

Comment:
Not `this:delete()`?

Reply:
`this:delete()` causes this:

![image](https://user-images.githubusercontent.com/60856959/149572325-3c7727fc-3df8-4a00-9234-9963de1ea463.png)

`this:hide()` is also used by dlg_config_world when opening up the ContentDB dialog so I just copied that.

Reply:
The current code will return to new world after installing the game, I think that's fine

One thing to note is that the game theme isn't applied when doing this